integration_test/myxql/test_helper.exs

```diff
@@ -97,7 +97,9 @@ excludes = [
   :add_column_if_not_exists,
   :remove_column_if_exists,
   # MySQL doesn't have a boolean type, so this ends up returning 0/1
-  :map_boolean_in_expression
+  :map_boolean_in_expression,
+  # MySQL doesn't support indexed parameters
+  :placeholders
```

Comment:
I think we should support placeholders on MySQL, we just won't actually send them as placeholders. WDYT?

Regarding the MySQL failures, please ignore them... unfortunately MySQL introduced breaking changes in the middle of their 8.x branch. :(

Reply:
Actually, let's keep MySQL raising for now. We can revisit later. :)